ocaml/xapi/xapi_globs.ml

```diff
@@ -1120,13 +1123,26 @@ let options_of_xapi_globs_spec =
   List.map
     (fun (name, ty) ->
       ( name
-      , (match ty with Float x -> Arg.Set_float x | Int x -> Arg.Set_int x)
+      , ( match ty with
+        | Float x ->
+            Arg.Set_float x
+        | Int x ->
+            Arg.Set_int x
+        | TimeSpanOfFloat x ->
+            Arg.Float
+              (fun y ->
+                let y' = Float.to_int (1000. *. y) in
+                x := Mtime.Span.(y' * ms)
```

Comment:
If you can review another pull request first and wait for it to be merged, you will be able to use `Clock.Timer.s_to_span` here, and `Clock.Timer.span_is_shorter elapsed ~than:!Xapi_globs.host_assumed_dead_interval` further up.

Reply:
Yeah. I can wait for that PR getting merged.